session.go

```diff
@@ -1584,6 +1628,32 @@ func (t *traceWriter) Trace(traceId []byte) {
 	}
 }
 
+type ObserveQuery struct {
```

Comment:
I had meant this the other way around, this is fine to be called `ObservedQuery`